jina/helper.py

```diff
@@ -1589,3 +1590,10 @@ def _parse_url(host):
         port = None
 
     return scheme, host, port
+
+
+async def is_port_free(host, port):
```

Comment:
why is it async?

Reply:
my bad it used to await co routine inside but not anymore